layouts/home.html

```diff
@@ -72,111 +72,108 @@ <h1 class="text-5xl font-medium">{{ .Title }}</h1>
           )
         ) }}
         {{ partial "docs-section.html" (dict
-          "Title" "Redis Enterprise Software"
-          "Description" "A self-managed data platform that unlocks the full potential of Redis at enterprise scale."
-          "ButtonLink" "./operate/rs"
-          "ButtonLabel" "Read more"
-          "LinksLeftTitle" "Learn more"
+            "Title" "Ingest and sync data to Redis"
+          "Description" "Enable pipeline sync of Redis Software with live data from your disk-based databases"
+	  "ButtonLink" "./integrate/redis-data-integration/quick-start-guide/"
+          "ButtonLabel" "Redis Data Integration (RDI) pipeline quickstart"  
           "LinksLeft" (slice
-            (dict "Text" "Quick start" "URL" "./operate/rs/installing-upgrading/quickstarts/redis-enterprise-software-quickstart/")
-            (dict "Text" "Clusters" "URL" "./operate/rs/clusters/")
-            (dict "Text" "Databases" "URL" "./operate/rs/databases/")
-	    (dict "Text" "Networking" "URL" "./operate/rs/networking/")
+            (dict "Text" "RDI feature overview" "URL" "./integrate/redis-data-integration/")
           )
         ) }}
         {{ partial "docs-section.html" (dict
-          "Title" "Redis Enterprise for Kubernetes"
-          "Description" "Redis Enterprise deployed on containerized software platforms and integrated with our partners."
-          "ButtonLink" "./operate/kubernetes/"
-          "ButtonLabel" "Read more"
-          "LinksLeftTitle" "Learn more"
-          "LinksLeft" (slice
-            (dict "Text" "Architecture" "URL" "./operate/kubernetes/architecture/")
-            (dict "Text" "Deployments" "URL" "./operate/kubernetes/deployment/")
-            (dict "Text" "Clusters" "URL" "./operate/kubernetes/re-clusters/")
-            (dict "Text" "Databases" "URL" "./operate/kubernetes/re-databases/")
+          "Title" "Monitor Redis with Prometheus"
+          "Description" "Get started with the Redis Software integration for Prometheus and Grafana"
+	  "ButtonLink" "./integrate/prometheus-with-redis-enterprise/"
+          "ButtonLabel" "Prometheus and Grafana with Redis guide"  
+	"LinksLeft" (slice
+            (dict "Text" "Prometheus and Graphana with Redis Cloud guide" "URL" "./integrate/prometheus-with-redis-cloud/")
           )
         ) }}
+	{{ partial "docs-section.html" (dict
+	  "Title" "Client tools to connect to a Redis server"
+          "Description" "You can use the CLI, Redis Insight, or the Redis VSCode extension to connect to a Redis server"
+	  "ButtonLink" "./develop/tools/"
+          "ButtonLabel" "Client tools overview guide"
+	  "LinksLeft" (slice
+	    (dict "Text"  "Java (asynchronous)" "URL" "./develop/clients/lettuce/")
+	 (dict "Text" "Go" "URL" "./develop/clients/go/")
+            (dict "Text" "PHP" "URL" "./develop/clients/php/")
+	    (dict "Text" "Spring Data" "URL" "./integrate/spring-framework-cache/")
+	    (dict "Text" "More client APIs" "URL" "./develop/clients/")
```

Comment:
These are actually the other client libraries, not the tools you refer to in the main link. See my other comment above for my thoughts about moving all the client library links to the same tile (and missing a couple out strategically). In this tile, I think we should link directly to the tools' pages.
```suggestion
	    (dict "Text"  "Redis CLI" "URL" "./develop/tools/cli/")
	    (dict "Text"  "Redis Insight" "URL" "./develop/tools/insight/")
	    (dict "Text"  "Redis for VSCode" "URL" "./develop/tools/redis-for-vscode/")	    
```